client/analysis.html

```diff
@@ -17,8 +17,85 @@
+                            to analyze comments returned to me from the
+                            <a href="https://developers.google.com/youtube/v3" target="_blank">Youtube Data API</a>.
+                        </p>
+                        <h3>Method</h3>
+                        <p>
+                            In order to ensure the most accurate analysis while handling the minimum amount
+                            of data, this tool takes 20 random comments from the Youtube API, aggregates them
+                            into a single string of sentences, and runs the chunk through both the NLP and
+                            Perspective APIs. Here is what each API is responsible for:
+                        </p>
+                        <!--Potentially, we will take the most popular comments.-->
+                        <p>
+                            NLP
+                            <ul>    
+                                NLP returns a magnitude and score value which give us the Overall Response. 
+                                The score value contributes to the corresponding adjective:
+                                Positive, Negative, or Mixed. If a video is "Clearly" one
+                                of these values, NLP returned a high magnitude rating and thus is confident 
+                                that the given data supports its score. Click <a href="https://cloud.google.com/natural-language/docs/basics#interpreting_sentiment_analysis_values" target="_blank">
+                                here</a> for the official documentation.
+                            </ul>
+                        </p>
+                        <p>
+                            Perspective
+                            <ul>    
+                                Perspective returns a series of probabilities which correspond to attributes such
+                                as Toxicity, Insult, Identity Attack, etc. These probabilities indicate how likely
+                                the text is to have these effects on the readers. It is important to note that all
+                                attributes except for Toxicity are in the experimental phase right now. Click <a href="https://github.com/conversationai/perspectiveapi/blob/master/2-api/models.md#all-attribute-types" target="_blank">
+                                here</a> for further explanation. 
+                            </ul>
+                        </p>
+                        <h3>Programming techniques</h3>
+                        <p>
+                            I've implemented techniques like caching, pattern-matching, 
+                            search, and string manipulation to provide the best possible experience
+                            and just to have some fun with this. 
+                            The code is all open source and can be found <a href="https://github.com/googleinterns/step99-2020" target="_blank">
```

Comment:
Tip: if you add `<base target="_blank">` to your `<head>`, then all links on the page will default to `target="_blank"` so you don't have to specify it.

Reply:
+1